--- a/src/components/settings/tabs/plugins/PluginModal.tsx
+++ b/src/components/settings/tabs/plugins/PluginModal.tsx
@@ -256,7 +256,7 @@ export default function PluginModal({ plugin, onRestartNeeded, onClose, transiti
                                 )}
                             </Tooltip>
                         ) : <div />}
-                        {!pluginMeta.userPlugin && (
+                        {!(pluginMeta.userPlugin || pluginMeta.folderName.endsWith("recar")) && (
                             <div className={cl("links")}>
                                 <FavoriteButton
                                     isFavorite={pluginSettings.isFavorite ?? false}
